tests/integration/data_sources_and_expectations/expectations/test_expect_column_distinct_values_to_be_in_set.py

```diff
@@ -77,25 +80,17 @@ def test_data_is_subset(batch_for_datasource: Batch) -> None:
     assert result.success
 
 
-@pytest.mark.xfail(strict=True)
-@parameterize_batch_for_data_sources(
-    data_source_configs=JUST_PANDAS_DATA_SOURCES, data=ONES_AND_TWOS
-)
-def test_empty_value_set(batch_for_datasource: Batch) -> None:
-    """Failing test that seems like a (pretty minor) bug"""
-    expectation = gxe.ExpectColumnDistinctValuesToBeInSet(column=COL_NAME, value_set=[])
-    result = batch_for_datasource.validate(expectation)
-    assert not result.success
-
-
-@parameterize_batch_for_data_sources(
-    data_source_configs=JUST_PANDAS_DATA_SOURCES, data=ONES_AND_TWOS
+@pytest.mark.unit
+@pytest.mark.parametrize(
+    "value_set",
+    [
+        pytest.param([], id="empty"),
+        pytest.param(None, id="none"),
+    ],
 )
-def test_value_set_is_none(batch_for_datasource: Batch) -> None:
-    # why do we even allow this?!?
-    expectation = gxe.ExpectColumnDistinctValuesToBeInSet(column=COL_NAME, value_set=None)
-    result = batch_for_datasource.validate(expectation)
-    assert result.success
+def test_bad_value_set(value_set: list | None) -> None:
```

Comment:
This changes previous behavior right?

Reply:
yes we no longer support falsy inputs - does that constitute a breaking change?